chain/events/filter/index_test.go

```diff
@@ -74,7 +74,7 @@ func TestEventIndexPrefillFilter(t *testing.T) {
 
 	dbPath := filepath.Join(workDir, "actorevents.db")
 
-	ei, err := NewEventIndex(dbPath)
+	ei, err := NewEventIndex(context.TODO(), dbPath, nil)
```

Comment:
For tests, just use `context.Background()`. Use `context.TODO()` for code where we actually plan on replacing it with a real context.